go/sgauth/internal/const.go

```diff
@@ -14,21 +14,11 @@
 // limitations under the License.
 package internal
 
-import "golang.org/x/net/context"
-
-// TokenSource supplies PerRPCCredentials from an oauth2.TokenSource.
-type GrpcApiKey struct {
-	Value string
-}
-
-// GetRequestMetadata gets the request metadata as a map from a TokenSource.
-func (key GrpcApiKey) GetRequestMetadata(ctx context.Context, uri ...string) (map[string]string, error) {
-	return map[string]string{
-		"x-goog-api-key": key.Value,
-	}, nil
-}
-
-// RequireTransportSecurity indicates whether the credentials requires transport security.
-func (key GrpcApiKey) RequireTransportSecurity() bool {
-	return true
-}
+// Keys for HTTP google headers
+const (
+	headerAuth = "authorization"
+	headerApiKey = "X-Goog-Api-Key"
+	headerQuotaUser = "X-Goog-QuotaUser"
```

Comment:
Quota-User

Reply:
go/system-parameters documented as "X-Goog-QuotaUser", I will correct the documentation.

Reply:
The latest version is already corrected to Quota-User. It is just not released yet.